### PR TITLE
docs(getting-started): update stackablectl op install output

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
```[tasklist]
### Run getting_started.sh via
- [x] `stackablectl`
- [x] `helm`
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- markdownlint-disable MD041 -->
+<!-- markdownlint-disable MD041 --><!-- markdownlint-disable MD041 -->
```
Comment (maltesander) [inline README.md:1]: Why twice :) ?

Comment (NickLarsenNZ) [reply README.md:1]: Same explanation as here: https://github.com/stackabletech/hdfs-operator/pull/553#discussion_r1686219526

**Review by maltesander — APPROVED** [review]
LGTM!

**Review by maltesander — APPROVED** [review]
LGTM!

**Pull request merged**